htdocs/core/modules/modDataPolicy.class.php

```diff
@@ -132,7 +132,7 @@ public function __construct($db)
 			array('DATAPOLICY_ADHERENT', 'chaine', '', $langs->trans('NUMBER_MONTH_BEFORE_DELETION'), 0),
 		);
 
-		$country = explode(":", $conf->global->MAIN_INFO_SOCIETE_COUNTRY);
+		if (isset($conf->global->MAIN_INFO_SOCIETE_COUNTRY)) $country = explode(":", $conf->global->MAIN_INFO_SOCIETE_COUNTRY);
```

Comment:
if $conf->global->MAIN_INFO_SOCIETE_COUNTRY is not defined, the variable $country is undefined when it should be defined to empty array.